src/App.css

```diff
@@ -486,3 +486,8 @@ padding: 0;
 .image-gallery-image video {
     padding-bottom: 3rem;
 }
+
+.showAllButton {
```

Comment:
would you mind moving this back to `FilterCheckboxes`? Since all the other styling is there